certbot/tests/util.py

```diff
@@ -169,6 +170,26 @@ def patch_get_utility(target='zope.component.getUtility'):
     return mock.patch(target, new_callable=_create_get_utility_mock)
 
 
+def patch_get_utility_with_stdout(target='zope.component.getUtility',
+                                  stdout=six.StringIO()):
+    """Patch zope.component.getUtility to use a special mock IDisplay.
+
+    The mock IDisplay works like a regular mock object, except it also
+    also asserts that methods are called with valid arguments.
```

Comment:
nit: I'd also add a sentence describing the behavior of messages written to stdout here.

Reply:
Added a line about argument `stdout` in the docstring.